Source/Dafny/Verifier/Translator.cs

```diff
@@ -11422,7 +11428,13 @@ public enum K { Free, Checked, Both }
 
     private bool CanSafelyInline(FunctionCallExpr fexp, Function f) {
       var visitor = new TriggersExplorer();
-      visitor.Visit(f);
+      if (f.Body != null) {
+        var body = f.Body;
+        if (f is PrefixPredicate pp) {
```

Comment:
I probably didn't give it much though. If I did, I'd say I used the short name `pp` since the name is short-lived and thus seemed more readable.

I would like it if C# changed the type of `f` inside the branch, but there's no way to do that and keep the same name, right?

Reply:
> I would like it if C# changed the type of f inside the branch, but there's no way to do that and keep the same name, right?

No I don't think so. I imagine such a design would create problems for mutable variables since inside the `if`, when read before a write `f` is a `PrefixPredicate`, but when writing to it `f` is still a `Function`.